### PR TITLE
Add paginationjs@2.0.7 w/ git auto-update

### LINKED ISSUE #9483: [Request] pagination.js
**Library name: pagination.js**
**Git repository url: https://github.com/superRaytin/paginationjs**
**npm package url(optional):** 
**License(s):**
**Official homepage: http://paginationjs.com/**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #9483 
Related issue(s): # #9414 

@kennynaoh Please review this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/superRaytin/paginationjs
 * Official website (optional, not the repository): http://paginationjs.com/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 178
   - Count of watchers: 16
   - Count of forks: 116
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9483, cc #9414, cc @superRaytin

### CODE REVIEW

**Comment by superRaytin** [comment]
👍 ❤️

**Comment by kennynaoh** [comment]
@hare1039 , Sorry for late response. Please rebase to the latest master branch. Thanks!

**Comment by hare1039** [comment]
@kennynaoh I have rebased to the latest master branch. Please review, thanks.

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by PeterDaveHello** [comment]
@kennynaoh doesn't `less` need to be excluded? https://github.com/superRaytin/paginationjs/blob/2.0.7/dist/pagination.less

**Comment by kennynaoh** [comment]
@PeterDaveHello Yes, it should be excluded.

File: ajax/libs/paginationjs/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "homepage": "http://paginationjs.com",
+  "filename": "pagination.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/superRaytin/paginationjs.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```
Comment (kennynaoh) [inline ajax/libs/paginationjs/package.json]: @hare1039 Please change the rule to exclude the `less` file, thanks!

Comment (hare1039) [inline ajax/libs/paginationjs/package.json]: @kennynaoh Added the rule "**/!(*.less)" to exclude .less. Please review.

**Comment by kennynaoh** [comment]
LTGM, ping @cdnjs/intern2

**Comment by PeterDaveHello** [comment]
@kennynaoh you should also approve this in PR review feature.

**Comment by PeterDaveHello** [comment]
Thanks for your contribution.

**Pull request merged**